exemplar/build.gradle

```diff
@@ -32,11 +32,21 @@ jar {
   manifest {
     attributes 'Main-Class': 'com.squareup.exemplar.ExemplarServiceKt'
   }
-
-  // This line of code recursively collects and copies all of a project's files
-  // and adds them to the JAR itself. One can extend this task, to skip certain
-  // files or particular types at will
-  from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+  classifier = 'unshaded'
 }
 
 apply from: rootProject.file('gradle/gradle-mvn-push.gradle')
+apply plugin: 'com.github.johnrengelman.shadow'
+
+task shadedJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
```

Comment:
nit: should the task be named shadowJar? not sure what the convention is for gradle. shaded might be a maven term

Reply:
Sure, I'll rename. I just made the assumption was a Java "thing" people knew.

Reply:
Actually, renaming this shadowJar would collide with the shadowJar task from the plugin. Just going to leave as-is. Since this is being published to Maven anyway, it's probably ok to reuse some terminology here.

Reply:
ok. i'd probably go with something like shadowJarTask